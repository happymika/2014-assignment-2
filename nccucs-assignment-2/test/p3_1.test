# p3 -- Simple test 1.

# Create a graph
CreateGraph Taiwan

# Add nodes
CreateNode TaipeiToTaichung 10
CreateNode TaichungToKaohsiung 15
CreateNode KaohsiungToTaipei 20
CreateNode KaohsiungToPingtung 3
AddNode Taiwan TaipeiToTaichung
AddNode Taiwan TaichungToKaohsiung
AddNode Taiwan KaohsiungToTaipei
AddNode Taiwan KaohsiungToPingtung

# Add edges
AddEdge Taiwan TaipeiToTaichung TaichungToKaohsiung
AddEdge Taiwan TaichungToKaohsiung KaohsiungToTaipei
AddEdge Taiwan TaichungToKaohsiung KaohsiungToPingtung

# Print some stuff
ListNodes Taiwan
ListChildren Taiwan TaipeiToTaichung
ListChildren Taiwan TaichungToKaohsiung
ListChildren Taiwan KaohsiungToPingtung