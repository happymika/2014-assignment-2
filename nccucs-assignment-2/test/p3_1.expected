#p3 -- Simple test 1.

# Create a graph
created graph Taiwan

# Add nodes
created node TaipeiToTaichung with cost 10
created node TaichungToKaohsiung with cost 15
created node KaohsiungToTaipei with cost 20
created node KaohsiungToPingtung with cost 3
added node TaipeiToTaichung to Taiwan
added node TaichungToKaohsiung to Taiwan
added node KaohsiungToTaipei to Taiwan
added node KaohsiungToPingtung to Taiwan

# Add edges
added edge from TaipeiToTaichung to TaichungToKaohsiung in Taiwan
added edge from TaichungToKaohsiung to KaohsiungToTaipei in Taiwan
added edge from TaichungToKaohsiung to KaohsiungToPingtung in Taiwan

# Print some stuff
Taiwan contains: KaohsiungToPingtung KaohsiungToTaipei TaichungToKaohsiung TaipeiToTaichung
the children of TaipeiToTaichung in Taiwan are: TaichungToKaohsiung
the children of TaichungToKaohsiung in Taiwan are: KaohsiungToPingtung KaohsiungToPingtung
the children of KaohsiungToPingtung in Taiwan are: